RELEASE/scripts/autoscend/auto_providers.ash

```diff
@@ -1212,3 +1212,336 @@ boolean provideMoxie(int amt, boolean doEquips)
+			auto_buyUpTo(1, $item[Loose Purse Strings]);
+			autoCraft("smith", 1, $item[Lump of Brituminous Coal], $item[Loose purse strings]);
+		}
+		string max = "500meat " + (amt + 100) + "max";
+		if(speculative)
+		{
+			simMaximizeWith(loc, max);
+		}
+		else
+		{
+			addToMaximize(max);
+			simMaximize(loc);
+		}
+		delta = simValue("Meat Drop") - numeric_modifier("Meat Drop");
+		auto_log_debug("With existing and crafted gear we can get to " + result());
+		if(pass())
+			return result();
+	}
+	// Use limited resources like Inhaler
+	if(doEverything)
+	{
+		if(tryEffects($effects[
+		shadow waters, //200% meat, 100% item, 100% init, -10% combat
+		Sinuses For Miles, //200% meat
+		Car-Charged, //100% meat, 100% item, 5-10MP, 50% init, 50% spell dmg, +3 stats per fight
+		Incredibly Well Lit //100% meat, 50% item
+		]))
+			if(pass())
+				return result();
+		if(zataraAvailable() && (0 == have_effect($effect[Meet the Meat])) & auto_is_valid($effect[Meet the Meat]))
+		{
+			if(!speculative)
+			{
+				zataraSeaside("meat");
+			}
+			handleEffect($effect[Meet the meat]); //100% meat, 50% gear drops
+			if(pass())
+				return result();			
+		}
+		if((get_property("sidequestArenaCompleted") == "fratboy") && !get_property("concertVisited").to_boolean() && (have_effect($effect[Winklered]) == 0))
+		{
+			if(is_professor())
+			{
+				//Need to manually equip because professor
+				if(!have_equipped($item[beer helmet])) equip($item[beer helmet]);
+				if(!have_equipped($item[distressed denim pants])) equip($item[distressed denim pants]);
+				if(!have_equipped($item[bejeweled pledge pin])) equip($item[bejeweled pledge pin]);
+			}
+			else
+			{
+				outfit("Frat Warrior Fatigues");
+			}
+			if(!speculative)
+			{
+				cli_execute("concert 2"); //40% meat
+			}
+			handleEffect($effect[Winklered]); //40% meat
+			if(pass())
+				return result();
+		}
+		if(pass())
+			return result();
+		if(auto_monkeyPawWishesLeft() + auto_wishesAvailable() > 0)
+		{
+			boolean success = true;
+			int specwishes = 0;
+			while(auto_monkeyPawWishesLeft() + auto_wishesAvailable() > 0 && success)
+			{
+				foreach eff in $effects[Frosty, //200% meat, 100% item, 25 ML, 100% init
+				Braaaaaains, //200% meat, -50% item
+				Let's Go Shopping!,  //150% meat, 75% item, -300% myst
+				Always Be Collecting, //100% meat, 50% item
+				Incredibly Well Lit, //100% meat, 50% item
+				A View to Some Meat, //100% meat
+				Cravin' for a Ravin', //100% meat
+				Low on the Hog, //100% meat
+				Leisurely Amblin', //100% meat
+				Trufflin', //100% meat
+				Here's Some More Mud in Your Eye, //100% meat
+				Eau d' Clochard, //100% meat
+				Flapper Dancin', //100% meat
+				Fishing for Meat, //100% meat
+				Preternatural Greed] //100% meat
+				{
+					if(have_effect(eff) == 0)
+					{
+						if(!speculative)
+							success = auto_wishForEffect(eff);
+						specwishes +=1;
+						if(specwishes <= auto_monkeyPawWishesLeft() + auto_wishesAvailable())

```

Comment:
haven't ran this branch. I'd expect all wish sources are getting used up at nuns and maybe at wall of meat too. Is that happening? Is it what we want to happen?

Reply:
Wondering if we should reserve any

Reply:
I had the same thought tbh, but I also thought that doing nuns/wall of meat as quick as possible is probably desirable to reserve more adventures for other activities. It actually didn't use every single wish on my last run, I still had a pocket wish after the nuns.